src/scss/_vars.scss

```diff
@@ -95,44 +95,79 @@ $text-tokens: $button, $cta-sm, $cta-lg, $email, $label, $badge, $display0, $dis
 /// FONT SIZE TOKENS:
 ///////////////////////
 
-:root {
-  --heading-size0: 3.25rem;
-  --heading-size1: 2.44rem;
-  --heading-size2: 1.95rem;
-  --heading-size3: 1.56rem;
-  --heading-size4: 1.56rem;
-  --body-size1: 1.25rem;
-  --body-size2: 1rem;
-  --body-size3: 0.75rem;
-  --label-size1: 1.25rem;
-  --label-size2: 1rem;
-  --label-size3: 0.75rem;
+@mixin font-size($to, $name, $font-start, $line-start, $step: 0.25) {
```

Comment:
I like that you made this dynamically generated. Much easier to add new tokens if needed.